ui/push-health/TestFailure.jsx

```diff
@@ -31,24 +31,39 @@ export default class TestFailure extends React.PureComponent {
           </span>
         </Row>
         <div className="small">
-          <a
-            className="text-dark ml-3 px-1 border border-secondary rounded"
-            href={getJobsUrl({ selectedJob: jobId, repo, revision })}
-          >
-            {jobName}
-          </a>
-          <span className="ml-1">{config},</span>
-          <span className="ml-1">
-            {classification !== 'not classified' && (
-              <FontAwesomeIcon icon={faStar} />
-            )}
-            <span className="ml-1">{classification}</span>
+          <span>
+            {platform} {config}:
           </span>
+          {jobs.map(job => (
+            <span className="mr-2">
+              <a
+                className="text-dark ml-3 px-1 border border-secondary rounded"
+                href={getJobsUrl({ selectedJob: job.id, repo, revision })}
+                title={jobName}
+              >
+                {jobSymbol}
+              </a>
+              <a
+                className="logviewer-btn"
+                href={getLogViewerUrl(job.id, repo)}
+                target="_blank"
+                rel="noopener noreferrer"
+                title="Open the Log Viewer for this job"
+              >
+                <img
+                  style={{ height: '18px' }}
+                  alt="Logviewer"
+                  src={logviewerIcon}
+                  className="logviewer-icon text-dark"
+                />
+              </a>
+            </span>
+          ))}
         </div>
         {!!logLines.length &&
           logLines.map(logLine => (
             <Row className="small text-monospace mt-2 ml-3" key={logLine}>
-              {logLine}
+              {logLine.subtest} {logLine.message}
```

Comment:
Going forward, we may want to tweak which information shows up here.  But I think this is correct for the first go.